src/vsr/grid_blocks_missing.zig

```diff
@@ -54,6 +59,7 @@ pub const GridBlocksMissing = struct {
     pub const RepairTable = struct {
         index_address: u64,
         index_checksum: u128,
```

Comment:
Should we remove `index_address` / `index_checksum` then? They are duplicated with `table_info`